packaging/fedora/charliecloud.spec

```diff
@@ -123,6 +123,7 @@ ln -s "${sphinxdir}/js"    %{buildroot}%{_pkgdocdir}/html/_static/js
 %{_libdir}/%{name}/*.sh
 %{_libdir}/%{name}/contributors.bash
 %{_libdir}/%{name}/version.txt
+%{_libdir}/%{name}/lark_parser.egg
```

Comment:
Is this egg file needed at all times? Or can we get rid of it after the `configure` and `make` steps of our build process?

Reply:
We are forcing users to use the embedded lark-parser so the egg file must be present in order for ch-image to work.